### PR TITLE
fix fetching `latin1` encoded data

### PR DESCRIPTION
**Pull request by ayrat555**
## Motivation

some JSONs contain latin1 encoded data which
fails to be decoded

## Changelog

- fix fetching `latin1` encoded data
- process json file in token uri
- separate file for token instances logs

### CODE REVIEW

**Comment by ayrat555** [comment]
@vbaranov the issue was in log level. it's fixed now

**Comment by vbaranov** [comment]
@ayrat555 CHANGELOG entry is missing

**Comment by ayrat555** [comment]
@vbaranov I added a CHANGELOG entry

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 - [#2470](https://github.com/poanetwork/blockscout/pull/2470) - Allow Realtime Fetcher to wait for small skips
 
 ### Fixes
+- [#2779](https://github.com/poanetwork/blockscout/pull/2779) - fix fetching `latin1` encoded data
+- [#2755](https://github.com/poanetwork/blockscout/pull/2755) - various token instance fetcher fixes
```
Comment (vbaranov) [inline CHANGELOG.md]: @ayrat555 it is a duplicate

**Pull request merged**